src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,57 @@
+package duke;
+
+import duke.task.Task;
+import duke.task.TaskList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Storage {
+
+    private static final String taskListTxt = "duke.txt";
```

Comment:
Perhaps the name of your constant variable should be capitalised? Maybe something like:
```suggestion
    private static final String TASKLIST_TXT = "duke.txt";
```